public/about/index.html

```diff
@@ -368,117 +368,130 @@ <h3 class="text-xl font-extrabold mt-3 ">Pete Keleher</h3>
                             </div>
                         </li>
                     <!-- end team member -->
-                    
+
+                    <!-- team member -->
+                    <li class="mt-6 sm:mt-0">
+                        <img src="https://rotational.io/img/team/danielle-maxwell.png" alt="Danielle Maxwell" class="rounded-t-xl">
+                        <div class="bg-[#ECF6FF] rounded-b-xl">
+                            <div class="px-6 pt-8 pb-5">
+                                <h4 class="text-base font-bold text-[#1D65A6]">Junior Distributed Systems Engineer</h4>
+                                <h3 class="text-xl font-extrabold mt-3 ">Danielle Maxwell</h3>
+                                <p class="mt-3 text-base font-extralight">WebAssembly Specialist</p>
+                            </div>
+                        </div>
+                    </li>
+                    <!-- end team member -->
+
```

Comment:
@masskoder should this be added to both `index.html` and `aboutus.html`? It seems a bit odd to have to add the same information to two different files